metadata/src/main/java/org/apache/kafka/controller/ActivationRecordsGenerator.java

```diff
@@ -165,6 +165,8 @@ static ControllerResult<Void> recordsForNonEmptyLog(
                         throw new RuntimeException("Should not have ZK migrations enabled on a cluster that was " +
                             "created in KRaft mode.");
                     }
+                    logMessageBuilder
+                            .append("since this is a de-novo KRaft cluster.");
```

Comment:
How about "This is expected because this is a de-novo KRaft cluster." ?

Reply:
This is better 👍 
I make a new commit to apply your comments.
Please take another look, when you have free time 🙇‍♂️

Reply:
@mumrah gently ping.
When you have free time, please take another look? 
Thanks in advance! 🙇‍♂️